lib/webauthn/public_key_credential_with_assertion.rb

```diff
@@ -31,5 +32,20 @@ def user_handle
     def raw_user_handle
       response.user_handle
     end
+
+    private
+
+    def appid_extension_output
+      return if client_extension_outputs.nil?
+
+      client_extension_outputs[:appid] || client_extension_outputs['appid']
+    end
+
+    def appid
+      appid_domain = relying_party.legacy_u2f_appid || relying_party.id ||
+                     relying_party.origin || raise("Unspecified expected origin")
+
+      URI.parse(appid_domain).to_s
```

Comment:
I don't think this is needed either: AppID should be a web origin – like `https://login.example.com` – so doing `URI.parse(appid).to_s` should always return `appid` if I'm not mistaken